tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
Removing babel and setting ts target to es6 means that you don't support non es6 compatible browser.

You may want to set this to es5 but be aware that ts doesn't transpile es6 to es5 out of the box (e.g https://github.com/Microsoft/TypeScript/issues/11209).
In case of errors, you may need to reintroduce babel after tsc.